readme.md

```diff
@@ -33,6 +33,9 @@ cryptoRandomString({length: 6, type: 'distinguishable'});
 cryptoRandomString({length: 10, type: 'ascii-printable'});
 //=> '`#Rt8$IK>B'
 
+cryptoRandomString({length: 10, type: 'alphanumeric'});
+//=> 'DMuKL8YtE7'
```

Comment:
index.d.ts and the readme should be in sync (please try to ensure this is the case for every PR).